cmd/cert-checker/main_test.go

```diff
@@ -103,7 +150,7 @@ func TestCheckCert(t *testing.T) {
 		},
 		NotBefore:             issued,
 		NotAfter:              goodExpiry.AddDate(0, 0, 1), // Period too long
-		DNSNames:              []string{"example-a.com", "foodnotbombs.mil"},
+		DNSNames:              []string{"example-a.com", "foodnotbombs.mil", "*.foodnotbombs.mil"},
```

Comment:
This will hit the forbiddenDomains stuff before the invalid character bit because of the `.mil` I think.

Reply:
Seems not to. See L184.